syntax = "proto3";
package cdm;
option java_package = "com.kitware.physiology.cdm";

message eScenario
{
  enum DecimalFormat
  {
    SystemFormatting  = 0;/*<<@brief Not provided, let the system do what it wants */
    DefaultFloat      = 1;
    FixedMantissa     = 2;
    SignificantDigits = 3;
  }


  enum DataRequestCategory
  {
    Patient            = 0;
    Physiology         = 1;
    Environment        = 2;
    GasCompartment     = 3;
    LiquidCompartment  = 4;
    ThermalCompartment = 5;
    TissueCompartment  = 6;
    Substance          = 7;
    AnesthesiaMachine  = 8;
    ECG                = 9;
    Inhaler            = 10;
  }
}
